FirebaseSwiftUI/FirebaseAuthSwiftUI/Sources/Services/AuthService.swift

```diff
@@ -82,17 +83,28 @@ public final class AuthService {
   public var authenticationFlow: AuthenticationFlow = .login
   public var errorMessage = ""
   public let passwordPrompt: PasswordPromptCoordinator = .init()
-
-  public var googleProvider: (any GoogleProviderAuthUIProtocol)?
-  public var facebookProvider: (any FacebookProviderAuthUIProtocol)?
-  public var phoneAuthProvider: (any PhoneAuthProviderAuthUIProtocol)?
+  private var googleProvider: (any GoogleProviderAuthUIProtocol)?
+  private var facebookProvider: (any FacebookProviderAuthUIProtocol)?
+  private var phoneAuthProvider: (any PhoneAuthProviderAuthUIProtocol)?
```

Comment:
These are private now because they shouldn't be accessed, should use `safeGoogleProvider`, etc.

Reply:
nit: I'd rather have the private apis be named `unsafeGoogleProvider` and the public ones plainly `googleProvider`

Reply:
Yeah, that's a better idea, done: [`a454559` (#1256)](https://github.com/firebase/FirebaseUI-iOS/pull/1256/commits/a454559be9e17f1bb7b01b59798b16adce686bbd)